### PR TITLE
Added Several Fextra Redirects

### PR DESCRIPTION
**Pull request by DrWhoFan13**
Added Redirects for:

- Fallout 4 & 76 Fextra wikis to Independent Fallout Wiki
- Minecraft Dungeons Fextra to the Minecraft Wiki
- TOTK Fextra to Zeldapedia
- Darkest Dungeon 2 Fextra to Darkest Dungeon Wiki.gg
- Hollow Knight Silksong Fextra to Hollow Knight Wiki
- Destiny Fextras to Destinypedia

### CODE REVIEW

**Review by KevinPayravi — CHANGES_REQUESTED** [review]
Thanks for the PR! Please make sure the wikis in each `origins` array are separated by commas. I recommend editing JSONs in an editor that does JSON validation to catch these sort of syntax errors.

**Comment by DrWhoFan13** [comment]
Done!

File: data/sitesEN.json [hunk 1]
```diff
@@ -1362,14 +1362,20 @@
...
         "origin_content_path": "/wiki/",
         "origin_main_page": "Darkest_Dungeon_Wiki"
-      }
+      },
+      {
+        "origin": "Darkest Dungeon 2 Wiki - Fextralife",
+        "origin_base_url": "darkestdungeon2.wiki.fextralife.com",
+        "origin_content_path": "/",
+        "origin_main_page": "Darkest+Dungeon+2+Wiki"
+      },
```
Comment (KevinPayravi) [inline data/sitesEN.json:1378]: Need to remove trailing commas after the last object in each array

**Review by KevinPayravi — CHANGES_REQUESTED** [review]

**Comment by DrWhoFan13** [comment]
Okay, _now_ it should be done.

**Review by KevinPayravi — APPROVED** [review]
Thank you!

**Pull request merged**